client/src/components/Restrictions.tsx

```diff
@@ -41,13 +55,14 @@ function Restrictions() {
     }
   };
 
+  useEffect(() => {
+    updateSelectedDaysToRedux(days);
+  }, [days]);
+
   return (
     <Section>
-      <Title title="3. Add Restrictions" data-test="title"></Title>
-      <Title
-        title="What days do you want to go to school?"
-        data-test="title"
-      ></Title>
+      <Title title="3. Add Restrictions"></Title>
+      <Title title="What days can you not go to school?"></Title>
```

Comment:
Would it make sense to change this to "What days do you not want to go to school?"?

Reply:
other than that, everything LGTM!

Reply:
Hmm yeah, this sounds better, I'll make those changes tmr morning and push so people can pull the latest.